screens/Budget/components/missingBudget.tsx

```diff
@@ -52,17 +36,47 @@ const MissingBudget = (
         })
     })
 
-    function createCopy() {
-        alert("Your nearset budget is " + (closetBudget !== undefined ? MONTHS_ORDER[parseInt(closetBudget.split('-')[1])] : 'none'))
+    const [copyBudget] = useMutation<CopyBudgetMutation>(CopyBudgetDocument, {
+        variables: { passwordHash, month, year, id: closetBudgetId },
+        onError: (error => {
```

Comment:
Let's be consistent with formatting our lambda functions and put parentheses around the error argument. Eg: `onError: ((error) => { ... })`